android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt

```diff
@@ -0,0 +1,22 @@
+package com.onegini.mobile.sdk.flutter
+
+enum class OneWelcomeWrapperErrors(val code: Int, val message: String) {
+    GENERIC(8000, "Something went wrong"),
+    USER_PROFILE_IS_NULL(8001, "User profile is null"),
+    AUTHENTICATED_USER_PROFILE_IS_NULL(8002, "Authenticated user profile is null"),
+    AUTHENTICATOR_NOT_FOUND(8004, "The requested authenticator is not found"),
+    HTTP_REQUEST_ERROR(8011, "OneWelcome: HTTP Request failed internally."),
+    ERROR_CODE_HTTP_REQUEST(8013, "OneWelcome: HTTP Request failed. Check Response for more info."),
+
+    // Errors that only occur on Android
+    AUTHENTICATOR_IS_NULL(8003, "Authenticator is null"),
+    IDENTITY_PROVIDER_NOT_FOUND(8005, "The requested identity provider is not found"),
+    QR_CODE_NOT_HAVE_DATA(8006, "QR-code does not have data"),
```

Comment:
Maybe we can make some of these errors a bit nicer :) `QR_CODE_NOT_HAVE_DATA` is not something that's nice to read and for `URL_IS_NOT_WEB_PATH` we already have a name which is widely adopted: "Malformed URL" see for instance. https://docs.oracle.com/javase/7/docs/api/java/net/MalformedURLException.html

Reply:
done